CONTRIBUTING.md

```diff
@@ -116,13 +116,13 @@ the community.
 
 This Code of Conduct is adapted from the [Contributor Covenant][homepage],
```

Comment:
@Programming-School-Pro-Coding This entire content is wrong. It is by mistake a code of conduct guide, not contributing one. We are replacing all the texts of this file with a correct contributing guide soon. You can close this PR.